src/SidePanel/dropdown/DropdownItem.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface DropdownItemProps {
+    title: string;
+    eventKey?: string;
+    onSelect?: (eventKey: string) => void;
+}
+
+const DropdownItem = ({ title, eventKey, onSelect }: DropdownItemProps) => {
+    const onClick = () => {
+        return onSelect(eventKey);
+    };
+
+    return (
+        <button type="button" className="dropdown-item" onClick={onClick}>
+            {title}
+        </button>
+    );
+};
```

Comment:
This could be shortened to:

```suggestion
const DropdownItem = ({ title, eventKey, onSelect }: DropdownItemProps) => (
        <button type="button" className="dropdown-item" onClick={() => onSelect(eventKey)}>
            {title}
        </button>
    );
```

Reply:
Generally copyright headers need to be added to all files.